config/manifests/bases/grafana-operator.clusterserviceversion.yaml

```diff
@@ -60,6 +61,15 @@ spec:
   links:
   - name: Grafana Operator
     url: https://grafana.github.io/grafana-operator
+  maintainers:
+  - email: pbraun@redhat.com
+    name: Peter Braun
+  - email: hstefans@redhat.com
+    name: Hubert Stefanski
+  - email: edvin.norling@gmail.com
+    name: Edvin Norling
+  - email: demtis.register@gmail.com
+    name: Igor Beliakov
```

Comment:
Hmm, no real issue with this, but maybe @ishanjainn or @theSuess know of a Grafana-owned mailing list that we could be added to? This would mean users would email `grafanaoperator@grafana.com` as opposed to individuals? Just a thought though, it's my corporate email anyway, so I don't mind :sweat_smile:

Reply:
Yeah this sounds good, I'll ask this to the IT team if they can add gmail ids to a group

Reply:
Agreed, and that would be great.

Reply:
We can now use -> `grafana-operator@grafana.com`